understanding/20/contrast-minimum.html

```diff
@@ -13,15 +13,14 @@ <h1>Understanding Contrast (Minimum)</h1>
       <h2>Intent of Contrast (Minimum)</h2>
       
       
-      <p>The intent of this Success Criterion is to provide enough contrast between text and
-         its background so that it can be read by people with moderately low vision (who do
-         not use contrast-enhancing assistive technology). For people without color deficiencies,
-         hue and saturation have minimal or no effect on legibility as assessed by reading
-         performance (Knoblauch et al., 1991). Color deficiencies can affect luminance contrast
-         somewhat. Therefore, in the recommendation, the contrast is calculated in such a way
-         that color is not a key factor so that people who have a color vision deficit will
-         also have adequate contrast between the text and the background.
-      </p>
+	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with moderately low vision.
+    This user group does not typically use contrast-enhancing assistive technology. 
+    For people without color deficiencies, hue and saturation have minimal or no effect on legibility as assessed by reading performance (Knoblauch et al., 1991).
+    Color deficiencies can affect luminance contrast somewhat. 
+    Therefore the contrast is calculated in such a way that color (hue) is not a key factor. 
+    People who have a color vision deficit (often referred to as "color blindness") will also have adequate contrast between the text and the background.</p> 
```

Comment:
maybe add a "do" to make it a bit clearer ("For people without color deficiencies ... People who do have a color vision deficit..."

"will also have adequate contrast between the text and the background." maybe at the end, add - to make it a more complete sentence - "if the requirements of this Success Criterion are followed", as currently it's left a bit hanging...

Reply:
Agree that this needs improvement.